Remove.py

```diff
@@ -1,35 +1,48 @@
 import PrintList
 
 
-def remove_1(to_Do, removed):
+def remove_1(to_Do):
```

Comment:
Why did we name this method `remove_1` instead of just `remove`? Numbering methods can get confusing in the long run. It's generally better to have more descriptive method names, even if they end up a _little_ longer.

Example: we have two methods which remove stuff. One removes the first item in a list. The other removes the last item in a list. Instead of `remove_1` and `remove_2` it'd be better to go with something like `remove_first_item` and `remove_last_item`.

Reply:
Named it remove, I cant remember why I had remove_1.